src/http/register-websocket/upgrade.js

```diff
@@ -16,14 +17,26 @@ module.exports = function upgrade (wss, { cwd, inventory, update }) {
     let lambda = get.ws('connect')
 
     // Create a connectionId uuid
-    let h = new Hashid
+    let h = new Hashid()
```

Comment:
I'm wondering if we shouldn't move all these calls into your new `request-id` lib (maybe rename that to `guid`). Related: `Date.now()` could in theory collide, maybe Date.now + random int?